src/Controller/Admin/AdminController.php

```diff
@@ -20,8 +25,22 @@ class AdminController extends AbstractController
      * @MenuItem(title="Overzicht", menu="admin", activeCriteria="admin_index", order=-1)
      * @Route("/", name="index", methods={"GET"})
      */
-    public function indexAction(): Response
+    public function indexAction(ActivityRepository $activitiesRepo, GroupRepository $groupsRepo): Response
     {
-        return $this->render('admin/index.html.twig');
+        if ($this->isGranted('ROLE_ADMIN')) {
+            $activities = $activitiesRepo->findBy([], ['start' => 'DESC']);
+        } else {
+            $user = $this->getUser();
+            assert($user instanceof LocalAccount);
+            $groups = $groupsRepo->findSubGroupsForPerson($user);
+            $activities = $activitiesRepo->findAuthor($groups);
+        }
+
+        // Only retain future activities
```

Comment:
you're retaining future and ongoing, correct?

Reply:
Good point!